### PR TITLE
Book tests

### PR DESCRIPTION
**Pull request by micaelacole**
# Summary
Unit tests for the Book and User models are created. The gems Database Cleaner, Faker and Factory Bot Rails are used together with RSpec.

In order to run the tests, the command `bundle exec rspec` is used.

# Trello Card
https://trello.com/c/SJso6Nga

### CODE REVIEW

File: spec/factories/users.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :user do
+    first_name { Faker::Name.first_name }
```
Comment (facucompe) [inline spec/factories/users.rb]: Correct the alignment

**Review by facucompe — DISMISSED** [review]
That small change

File: spec/factories/books.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :book do
+    genre       { Faker::Book.genre }
+    author      { Faker::Book.author }
+    cover_image { 'image.jpg' }
```
Comment (nicozar95) [inline spec/factories/books.rb]: Dont use string, use faker for this

**Review by nicozar95 — CHANGES_REQUESTED** [review]
Really small change

File: spec/factories/books.rb (hunk 2 above)
Comment (micaelacole) [reply spec/factories/books.rb]: I used a string since I couldn't find an appropriate Faker field for an image.

Comment (nicozar95) [reply spec/factories/books.rb]: Is the image it's a file we could fake a file 😉 https://github.com/stympy/faker/blob/master/doc/v1.9.1/file.md

Comment (nicozar95) [reply spec/factories/books.rb]: Or even better: https://github.com/stympy/faker/blob/master/doc/v1.9.1/avatar.md

Comment (micaelacole) [reply spec/factories/books.rb]: great! Thanks

File: spec/factories/users.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :user do
+    first_name  { Faker::Name.first_name }
```
Comment (nicozar95) [inline spec/factories/users.rb:3]: I dont think this is the correct indentation

**Review by nicozar95 — APPROVED** [review]

**Pull request merged**